packages/react-core/src/components/Nav/NavList.tsx

```diff
@@ -14,9 +14,9 @@ export interface NavListProps
   children?: React.ReactNode;
   /** Additional classes added to the list */
   className?: string;
-  /** Aria-label for the left scroll button */
+  /** Aria-label for the back scroll button */
```

Comment:
can you deprecate this and the `ariaRightScroll` prop and add `ariaBackScroll` and `ariaLeftScroll` props.